R/utils_tidy_data.R

```diff
@@ -89,9 +89,9 @@ tidy_cleaned_trips = function(cleaned_trips, project_crs = 4326, smallest_roundi
     # see: the section 'Re-using the placeholder for attributes' in
     # https://magrittr.tidyverse.org/
     {
-      sf::st_sf(.[, -"geometry"],
+      sf::st_sf(dplyr::select(., -geometry),
```

Comment:
Good catch. I removed a line that converted the data from data.frame to data.table, but did not consider the change here. `.[, -"colname"]` works on data.table objects.